publish_lambda.sh

```diff
@@ -322,7 +333,7 @@ def create_policy(publish_config: dict[str, Any]):
         assert isinstance(publish_config["s3-config-file"], str)
 
         bucket_name_and_object_key = publish_config["s3-config-file"].replace("s3://", "")
-        resource = f"arn:aws:s3:::{bucket_name_and_object_key}"
+        resource = f"arn:%awsOrGov%:s3:::{bucket_name_and_object_key}"
```

Comment:
this content ends up in `publish.py`, we don't execute `sed` on this content.
If I'm not wrong, you should use `${AWS_OR_AWS_GOV}` instead.
Please test :)

Reply:
Ahh I didn't realize this is going into publish.py! Thanks!! I just made the changes and tested them.